homeassistant/components/aqualogic/sensor.py

```diff
@@ -97,5 +97,7 @@ def async_update_callback(self):
                 self.async_write_ha_state()
             else:
                 self._attr_unit_of_measurement = SENSOR_TYPES[self._type][1][1]
+                self._attr_state = getattr(panel, self._type)
```

Comment:
Looks like we can simplify the if/else block to just be about the unit of measurement, since the other two lines are the same in both blocks.